### PR TITLE
Launcher: Invert the logic of skip opening of last workfile - OP-6968

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
This changes the initial toggle of the `Skip opening of last workfile` option in the Launcher, to inherit from the `ayon+settings://core/tools/Workfiles/last_workfile_on_startup` setting.

If an application is not part of the profiles, it'll initially be set to `Skip opening of last workfile`.

This does not influence existing settings in the launcher, where a user has enabled `Skip opening of last workfile`.

## Testing notes:
1. Test a variety of combinations for `ayon+settings://core/tools/Workfiles/last_workfile_on_startup` and verify the initial toggle of `Skip opening of last workfile` is set according to the profiles.
2. Test `Skip opening of last workfile` is actually skipping opening of the last workfile.
3. Test host without a profile in `ayon+settings://core/tools/Workfiles/last_workfile_on_startup`, which should initially have `Skip opening of last workfile` enabled.

NOTE: In order to test the initial toggle of `Skip opening of last workfile`, it might useful to remove the launcher settings at; `C:\Users\[user]\AppData\Local\ynput\ayon\launcher_tool.json`

### CODE REVIEW

File: client/ayon_core/tools/launcher/models/actions.py [hunk 1]
```diff
@@ -464,14 +510,18 @@ def _get_action_items(self, project_name):
             label = action.label or identifier
             variant_label = getattr(action, "label_variant", None)
             icon = get_action_icon(action)
+            host_name = identifier.replace("application.", "").split("/")[0]
```
Comment (iLLiCiTiT) [inline client/ayon_core/tools/launcher/models/actions.py:513]: This is not how host name works...

**Comment by tokejepsen** [comment]
@iLLiCiTiT the code changes looks good, but it does not save "Skip opening of last workfile" checkbox state.

**Review by tokejepsen — COMMENTED** [review]
@iLLiCiTiT the code changes looks good, but it does not save "Skip opening of last workfile" checkbox state.

**Comment by tokejepsen** [comment]
@iLLiCiTiT this is working now. Could you sign off on the code?

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**